src/SimpleAmqpClient/Channel.h

```diff
@@ -221,12 +287,12 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
 
   explicit Channel(const std::string &host, int port,
                    const std::string &username, const std::string &password,
-                   const std::string &vhost, int frame_max);
+                   const std::string &vhost, int frame_max, bool sasl_external = false);
```

Comment:
Don't use a default parameter. Update all callers instead, here and throughout.

Reply:
Done